lib/matplotlib/scale.py

```diff
@@ -1,16 +1,36 @@
 """
 Scales define the distribution of data values on an axis, e.g. a log scaling.
-They are defined as subclasses of `ScaleBase`.
 
-See also `.axes.Axes.set_xscale` and the scales examples in the documentation.
+Scales are subclasses `ScaleBase`. Their mapping is implemented through `.Transform`
```

Comment:
This is pretty technical for right at the top?  Maybe down around the new L26 would be better?